### PR TITLE
Parse an error to substitute a password value

### PR DESCRIPTION
**Pull request by StanislavTarasenko**
This PR closes [the security issue](https://github.com/kofile/ko-search-api/issues/433)

Another way is to remove configs from the Error
```js
new Error(`${error.message} — got \`${inspect(value)}\``) -> new Error(error.message)
```

! `error.message` contains all the problematic properties

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,15 @@ const clone = require('clone')
 
 const INVALID = 'invalid'
 
+const secureData = object => {
+  for (const prop in object) {
+    if (prop === 'password') object[prop] = '*****'
```
Comment (neezer) [inline index.js]: Ehh, I'm not a fan of hard-coding this value. What if I add a new prop named `bananas` that has sensitive information that I want to scrub? We'd have another PR on our hands.

I think we should accept a blacklist during initialization and match on items in that blacklist.

Comment (StanislavTarasenko) [reply index.js]: Agree! Also, we can use `pino` as a logger adapter and use [`pino-noir`](https://www.npmjs.com/package/pino-noir) to redact sensitive information from logs.

Comment (neezer) [reply index.js]: Oh, crap. Completely forgot what repo I was in. This is the wrong repo for this change. We should be adding the feature to https://github.com/kofile/log.

**Comment by neezer** [comment]
Oh, crap. Completely forgot what repo I was in. This is the wrong repo for this change. We should be adding the feature to https://github.com/kofile/log.

**Pull request closed without merging**